lib/components/term.js

```diff
@@ -38,6 +38,7 @@ export default class Term extends Component {
     this.term.prefs_.set('receive-encoding', 'raw');
     this.term.prefs_.set('send-encoding', 'raw');
     this.term.prefs_.set('alt-sends-what', 'browser-key');
+    this.term.prefs_.set('copy-on-select', false);
```

Comment:
For another PR, this could be an option. A lot of Linux users are used to copy on select from xterm